### ISSUE TITLE
[Feature request] Tab completion

### ISSUE THREAD

**Issue by Volgeop** [LogBlock/LogBlock#799]
Hi, could you add tab completion for sub commands?

**Comment by Brokkonaut** [comment]
Yes, tab completions would be great, but its a lot of work to add them, so I cannot tell when I will have time for it.
